MapboxCoreNavigation/Constants.swift

```diff
@@ -92,3 +92,8 @@ let FasterRouteFoundEvent = "navigation.fasterRoute"
  The number of seconds remaining on the final step of a leg before the user is considered "arrived".
  */
 public var RouteControllerDurationRemainingWaypointArrival: TimeInterval = 3
+
+/**
+ The minimum distance of a tunnel intersection on a given route step.
+ */
+public var RouteProgressMinimumTunnelIntersectionDistance: CLLocationDistance = 100
```

Comment:
This constant is never used inside of RouteProgress. In fact, it’s only used inside MapboxNavigation, for a UI-specific feature. So let’s move the constant to the other Constants.swift and give it a name like `MBMinimumTunnelLengthForUsingNightStyle`.